prompt/prompt-executor/prompt-executor-clients/prompt-executor-dashscope-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/dashscope/DashscopeParams.kt

```diff
@@ -0,0 +1,28 @@
+package ai.koog.prompt.executor.clients.dashscope
+
+import ai.koog.prompt.params.LLMParams
+
+/**
+ * DashScope-specific parameters for fine-tuning the model's behavior.
+ * These parameters are compatible with OpenAI API format.
+ *
+ * @property temperature Controls randomness: lower values for more focused, higher values for more creative outputs.
+ * @property maxTokens The maximum number of tokens to generate.
+ * @property toolChoice Controls which (if any) tool is called by the model.
+ */
+public data class DashscopeParams(
```

Comment:
`DashscopeParams` should inherit from `LLMParams`. But if it doesn’t extend `LLMParams`, then it’s better not to use it at all